### PR TITLE
chore(ci): extend phpunit matrix with mariadb

### PR DESCRIPTION
**Pull request by pweyck**

### CODE REVIEW

File: .github/workflows/integration.yml [hunk 1]
```diff
@@ -111,32 +111,35 @@ jobs:
...
           - name: integration
-            path: tests/integration/Elasticsearch
+            path: Elasticsearch
           - name: integration
-            path: tests/integration/Storefront
+            path: Storefront
           - name: migration,devops
+        db:
+          - mysql:8.0
+          - mariadb:10
```
Comment (pweyck) [inline .github/workflows/integration.yml]: mariadb:11 does not work. The container does not get healthy

Comment (shyim) [reply .github/workflows/integration.yml]: maybe because of your health check, i think the binaries are renamed in that version

**Review by philipreinken — APPROVED** [review]

File: .github/workflows/integration.yml (hunk 1 above)
Comment (pweyck) [reply .github/workflows/integration.yml]: `--health-cmd="mysqladmin ping || mariadb-admin ping"` did the trick

**Comment by pweyck** [comment]
I'm still not sure if we always want to execute the whole matrix. Maybe only execute the whole matrix in the nightly?

**Review by shyim — APPROVED** [review]

**Pull request merged**